tests/js/web/NavMainComponentTest.js

```diff
@@ -462,14 +462,30 @@ describe('NavMainComponent', function () {
             this.$moreIconLink.trigger(this.clickEvent2);
 
             expect(this.$html.find('.nav-tertiary').hasClass('is-open')).to.be.false;
-        });  
+        });
+
+        it('should close the quaternary nav if it is already open', function () {
+            this.$moreIconLink.trigger(this.clickEvent);
+            this.$tertiaryLink.trigger(this.clickEvent);
+            this.$moreIconLink.trigger(this.clickEvent2);
+
+            expect(this.$html.find('.nav-quaternary').hasClass('is-open')).to.be.false;
```

Comment:
Should we chuck in an ```expect(this.$html.find('.nav-quaternary').hasClass('is-open')).to.be.true;``` after the first click events? Wouldn't this test pass if the functionality that adds `'is-open` is broken? Admittedly that would (hopefully) fail elsewhere. Might be nice to catch that here also though, so it shouts very loudly if something is broken.